api/working-with-extensions/publishing-extension.md

```diff
@@ -212,14 +212,25 @@ Verifying your publisher will:
 
 ![Verified publisher indicators in VS Code](images/publishing-extension/verified-publishers.png)
 
+To verify a publisher:
```

Comment:
The above sentence should be removed now "Verifying your publisher will" it looks duplicated

Reply:
Done 👍